src/frontend/admin/src/components/presentational/button/CloseNotificationButton.tsx

```diff
@@ -1,14 +1,14 @@
 import CloseIcon from "@mui/icons-material/Close";
-import { closeSnackbar } from "notistack";
+import { closeSnackbar, SnackbarKey } from "notistack";
 import * as React from "react";
 
-export function CloseNotificationButton({ snackbarId }: any) {
+export function CloseNotificationButton(key: SnackbarKey) {
   return (
     <CloseIcon
       data-testid="close-notification"
       fontSize="small"
       sx={{ cursor: "pointer" }}
-      onClick={() => closeSnackbar(snackbarId)}
+      onClick={() => closeSnackbar(key)}
     />
   );
 }
```

Comment:
This diff looks not to be in the right fixup

Reply:
yes but that's what notistack typing wants